app/assets/stylesheets/components/_form.scss

```diff
@@ -70,3 +58,74 @@ input::-webkit-inner-spin-button {
   -webkit-appearance: none;
   margin: 0;
 }
+
+
+// wtf-forms.css
+.checkbox,
+.radio {
+  cursor: pointer;
+  display: inline-block;
+  padding-left: $radio-checkbox-space;
+  position: relative;
+}
+
+.checkbox input,
+.radio input, {
+  opacity: 0;
+  position: absolute;
+  z-index: -1;
+}
+
+// scss-lint:disable VendorPrefix
+.indicator {
+  background-color: $gray-light;
```

Comment:
i think we want these to have a background that matches the form field background with a 1px border of darker blue; i.e., 
<img width="73" alt="radio" src="https://cloud.githubusercontent.com/assets/1060893/17940821/caa49488-69fe-11e6-81ea-d0acb01cad2f.png">